vimiv/api/settings.py

```diff
@@ -493,9 +493,19 @@ class title:  # pylint: disable=invalid-name
 class metadata:  # pylint: disable=invalid-name
     """Namespace for metadata related settings."""
 
+    # Default sets
+    defaults = [
+        "Make, Model, DateTime, ExposureTime, FNumber, IsoSpeedRatings, FocalLength, LensMake, LensModel, PixelXDimension', 'PixelYDimension, ExposureBiasValue",  # pylint: disable=line-too-long,useless-suppression
```

Comment:
Are the single quotes around `PixelXDimension` and `PixelYDimension` intentional?

Reply:
You are right, they do not make any sense. However, I think I can remove both these key altogether. I expected them to display the resolution of the image, but they do not seem to work.